Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM node:6-onbuild
 MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
-RUN npm run install:clean --production
```

Comment:
i'm not sure if this is ok. It didn't make sense to clean out the node modules every time. I can change it back if it's wrong.